### PR TITLE
Update 0.9.0

### PR DESCRIPTION
**Pull request by Zakrok09**
# Update 0.9.0

Update fixes:

1. #6 
2. #8 
3. #7 
4. #5 

While also: #9 

###### I need to learn how to write good PRs

### CODE REVIEW

File: src/lib/translations/translations.js [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
-        "homepage.fourth.5": "does not contain any pirated content",
-        "homepage.fourth.6": "Send us an email",
-        "homepage.fourth.7": "Open an issue"
+        "homepage.fourthSection.question": "Found a missing link",
+        "homepage.fourthSection.listTitle": "If the link we are missing",
+        "homepage.fourthSection.item1": "is a legitimate source of information",
+        "homepage.fourthSection.item2": "is the official website of e.g. shops for texbooks and",
+        "homepage.fourthSection.item3": "does not contain any pirated content",
+        "homepage.fourthSection.sendEmail": "Send us an email",
+        "homepage.fourthSection.openIssue": "Open an issue"
```
Comment (dnbln) [inline src/lib/translations/translations.js:23]: Still not good enough...

https://lokalise.com/blog/translation-keys-naming-and-organizing/#2_Give_translation_keys_meaningful_names

File: src/routes/links/+page.svelte [hunk 2]
```diff
@@ -1 +1,159 @@
...
+    export let data;
+    let categories = data.links.categories;
+    let boxView = true;
+    let buttontext = "📅 Boxes";
+
+    function handleViewToggle() {
+        boxView = !boxView;
+        
+        buttontext = boxView ? '📅 Boxes' : '📄 Compact';
+    }
```
Comment (dnbln) [inline src/routes/links/+page.svelte:15]:
```suggestion
    }

    $: {
        buttontext = boxView ? '📅 Boxes' : '📄 Compact';
    }
```

File: src/routes/links/+page.svelte [hunk 3]
```diff
@@ -1 +1,159 @@
...
+
+    function handleToggleExtra(section) {
+        categories[section].showExtra = !categories[section].showExtra;
+    }
+</script>
+
+<div class="abovecategories">
+    <div class="filterButtons modeChoice">
+        <button class="filterButton" on:click={() => handleViewToggle()}>{buttontext}</button>
```
Comment (dnbln) [inline src/routes/links/+page.svelte:28]:
```suggestion
        <button class="filterButton" on:click={handleViewToggle}>{buttontext}</button>
```

File: src/lib/Reusable/Linklistitem.svelte [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+<div class="listItem">
+    <h3>{title}</h3>
+    <p>{desc}</p>
+    <div class="tags">
+        {#each tags as tag}
+            <span class="tag" style="background-color: {tag.color};">{tag.text}</span>
+        {/each}    
+    </div>
+    <a href={href} target="_blank">{anchorText}</a>
```
Comment (dnbln) [inline src/lib/Reusable/Linklistitem.svelte:17]:
```suggestion
    <a {href} target="_blank">{anchorText}</a>
```

File: src/lib/Reusable/Linkpagebox.svelte [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+
+<div class="Linkbox">
+    <!-- <div class="blackbar"></div> -->
+    <div class="background" style="background-image: url('/background/links/{tags[0].text}.svg');">
+        <h3>{title}</h3>
+    </div>
+    <p>{desc}</p>
+    <!-- <img {src} {alt}> -->
+    <a {href} target="_blank" class="AnchorBox ThirdBoxLink"> {anchorText} </a>
```
Comment (dnbln) [inline src/lib/Reusable/Linkpagebox.svelte:19]: Should there be spaces around the `{anchorText}`?

File: src/lib/data/non-stores.js [hunk 6]
```diff
@@ -0,0 +1 @@
+// to be used
```
Comment (dnbln) [inline src/lib/data/non-stores.js:1]: What's this file for?

Comment (Zakrok09) [reply src/lib/data/non-stores.js:1]: I made it to store json in it for some reason

File: src/routes/links/+page.svelte (hunk 2 above)
Comment (Zakrok09) [reply src/routes/links/+page.svelte:15]: Is it required to make it a reactive?

File: src/lib/translations/translations.js (hunk 1 above)
Comment (Zakrok09) [reply src/lib/translations/translations.js:23]: Will be fixed next time again

File: src/routes/links/+page.svelte (hunk 2 above)
Comment (dnbln) [reply src/routes/links/+page.svelte:15]: Cleans up the code a little bit, not really required.

**Pull request merged**